lib/active_data/model/associations/reflections/references_many.rb

```diff
@@ -0,0 +1,58 @@
+require 'active_data/model/associations/reflections/base'
+
+module ActiveData
+  module Model
+    module Associations
+      module Reflections
+        class ReferencesMany < Base
+          def macro
```

Comment:
Don't need this method anymore here (https://github.com/pyromaniac/active_data/blob/master/lib/active_data/model/associations/reflections/base.rb#L17-L19)